packages/mandelbrot/assets/scss/components/_file-browser.scss

```diff
@@ -31,36 +18,37 @@
     font-weight: bold;
 }
 
-.select2-selection.select2-selection--single {
-    border-color: $color-frame-border;
-
-    .select2-container--focus & {
-        --border-color: rgb(var(--skin-links));
-
-        outline: none;
-        border-color: $color-link;
-        border-color: var(--border-color, $color-link);
-    }
-}
-
-.select2-results__option {
-    @include font(caption);
-
+.FileBrowser-select {
+    position: relative;
+    appearance: none;
     min-width: 15em;
-
-    &[aria-selected="true"] {
-        background-color: rgba($color-frame-border, 0.1) !important;
+    padding: 0 24px 0 8px;
+    border: 1px solid rgba(83, 83, 99, 0.25);
+    border-radius: 4px;
+    font-size: 0.875rem;
+    color: #444;
+    line-height: 28px;
```

Comment:
It‘s not perfectly centered for me on Firefox, seems that `1` without unit have a better result in all browsers I have tested. Or actually no value at all, I think browsers vertically center text in native select by default.

Reply:
Removing it completely did not seem to affect it at all 👍